.github/workflows/windows-build-and-test.yaml

```diff
@@ -115,18 +124,22 @@ jobs:
 
     # Force install PostgreSQL 17 since the package still on moderation
     # https://community.chocolatey.org/packages/postgresql17
-    - name: Install PostgreSQL ${{ matrix.pg }}
-      if: steps.cache-postgresql.outputs.cache-hit != 'true'
+    - name: Install PostgreSQL ${{ matrix.pg }} (using ${{ matrix.pg_version }})
+      if: github.event_name != 'schedule' && steps.cache-postgresql.outputs.cache-hit != 'true'
       run: |
         choco feature disable --name=usePackageExitCodes
         choco feature disable --name=showDownloadProgress
-        if(${{ matrix.pg }} -eq 17) {
-          choco install postgresql${{ matrix.pg }} --version 17.0.0 `
-            --force -y --install-args="'--prefix $HOME\PostgreSQL\${{ matrix.pg }} --extract-only yes'"
-        } else {
-          choco install postgresql${{ matrix.pg }} `
-            --force -y --install-args="'--prefix $HOME\PostgreSQL\${{ matrix.pg }} --extract-only yes'"
-        }
+        choco install postgresql${{ matrix.pg }} --version ${{ matrix.pg_version }} `
+          --force -y --install-args="'--prefix $HOME\PostgreSQL\${{ matrix.pg }} --extract-only yes'"
+
+    # This is for nightly builds. Here we pick the latest version of the package.
+    - name: Install PostgreSQL ${{ matrix.pg }} (using ${{ matrix.pg_version }})
```

Comment:
```suggestion
    - name: Install PostgreSQL ${{ matrix.pg }}
```

Reply:
I wanted to make the exact version used very clear in the CI run, so that you do not have to dig through the code if there is a failure, hence added to the title. Do you think this would be problematic?

Reply:
Never mind. I noted this was for the nightly build.